src/sentry/uptime/consumers/results_consumer.py

```diff
@@ -91,7 +91,15 @@ def handle_result_for_project(
             "status": result["status"],
             "mode": ProjectUptimeSubscriptionMode(project_subscription.mode).name.lower(),
         }
-        metrics.incr("uptime.result_processor.handle_result_for_project", tags=metric_tags)
+
+        status_reason = "none"
+        if result["status_reason"]:
+            status_reason = result["status_reason"]["type"]
```

Comment:
Should we just shove it into `metric_tags`?

Reply:
We could... it didn't seem relevant to the other stats though